src/aiovodafone/api.py

```diff
@@ -201,6 +203,37 @@ async def get_voice_data(self) -> dict[Any, Any]:
     async def logout(self) -> None:
         """Router logout."""
 
+    @abstractmethod
+    async def traceroute(
```

Comment:
I think you can safely move those in the specific Technicolor API as Sercomm doesn't support those.

Reply:
done.